### PR TITLE
Update/loading placeholder

### PR DESCRIPTION
**Pull request by louise-r-blue**
This is a nice clean update to the placeholder behaviour on the search results page. Thanks to @Rabid-Dan 

Main changes include:
- The placeholder class is now contained and positioned absolute in a span element in the search results card div.
- Staggering the removal of the placeholder by using the css opacity property in the window on load function.
- Adding further styles to the placeholder class that mimic closely the search results card.

### CODE REVIEW

File: app/assets/javascripts/search-result-load-in.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+        searchResultPlaceholder++;
       });
+    });
+
+    function resultTransition(searchResultItem, counter, transitionClass) {
+      searchResultItem
+        .css({
+          transitionDelay: (counter * 0.1) + 's',
+        })
+        .addClass(transitionClass);
```
Comment (ghostPepperWonton) [inline app/assets/javascripts/search-result-load-in.js:22]: Great DRY-ing 👏

File: app/assets/stylesheets/placeholder.scss [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
   background-size: // background-size maps to background-position
     80% 20%, // top line width + height
     80% 15%, // middle line
     80% 15%, // bottom line
     50% 25%; // bottom right square
+ transition: {
+    duration: .2s;
+    property: opacity;
+    timing-function: ease-in-out;
+  };
```
Comment (ghostPepperWonton) [inline app/assets/stylesheets/placeholder.scss]: We recently added a mixin for this called `transition`. Can you use that here?

**Review by ghostPepperWonton — COMMENTED** [review]
Suuuuper minor fix, but if you can get that mixin in, we're all good to go :)

**Review by Rabid-Dan — APPROVED** [review]
👏 G👏 R👏 E👏 A👏 T👏 !

**Review by ghostPepperWonton — APPROVED** [review]
✨

**Pull request merged**